vhdbuilder/packer/install-dependencies.sh

```diff
@@ -315,8 +317,10 @@ done
 #UNITE swift and overlay versions?
```

Comment:
where is dropgz in this file?

Reply:
it's in vhdbuilder/packer/components.json